core/src/tasks/record/makeExports.ts

```diff
@@ -16,6 +17,12 @@ export class GrouparooRecordsCheckReady extends CLSTask {
       ? process.env.GROUPAROO_DISABLE_EXPORTS !== "true"
       : true;
 
-    await RecordOps.makeReadyAndCompleteImports(limit, toExport);
+    const records = await RecordOps.makeExports(limit);
+
+    if (toExport) {
+      for (const record of records) {
+        await CLS.enqueueTask("record:export", { recordId: record.id });
+      }
+    }
```

Comment:
Previously, when `toExport=true` we would not create the exports in the db since the imports were going directly to `complete` instead of passing through `processing`. 

In this case we are now creating the exports, but just not enqueuing the task. I think this change is ok since we are still checking for `GROUPAROO_DISABLED_EXPORTS` within `records:reEnqueueStuckExportingRecords`, but wanted to note the change.